content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -1,21 +1,20 @@
 +++
-title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
-description = "Instructions for installing Kubeflow with kfctl_existing_arrikto.yaml config"
+title = "Multi-user, auth-enabled Kubeflow with kfctl_istio_dex"
+description = "Instructions for installing Kubeflow with kfctl_istio_dex.yaml config"
 weight = 4
 +++
 
 Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-This installation of Kubeflow is maintained by
-[Arrikto](https://www.arrikto.com/), it is geared towards existing Kubernetes
+This installation of Kubeflow is geared towards existing Kubernetes
 clusters and does not depend on any cloud-specific feature.
 
 In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and
 [Istio](https://istio.io/) for vendor-neutral authentication.
 
 This deployment works well for on-prem installations, where companies/organizations need LDAP/AD integration for multi-user authentication, and they don't want to depend on any cloud-specific feature.
 
-![kfctl_existing_arrikto_architecture](../../kfctl_existing_arrikto-architecture.svg)
+![kfctl_istio_dex_architecture](../../kfctl_istio_dex-architecture.svg)
 
 Read the relevant [article](https://journal.arrikto.com/kubeflow-authentication-with-istio-dex-5eafdfac4782) for more info about this architecture.
```

Comment:
Will this link also be relevant after the updates?

Reply:
Hello @NikeNano, I think it's relevant as it describes the approach taken for this configuration.

Reply:
Sounds good,I was not sure how big the differences would be between this and the original arrikto configurations.